Wrappers/Python/cil/io/RAWFileWriter.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
+#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   substantial contributions by UKRI-STFC and University of Manchester.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.import numpy as np
+
+from cil.framework import AcquisitionData, ImageData
+import os, re
+import sys
+from cil.framework import AcquisitionData, ImageData
+import re
+from cil.io import utilities
+import configparser
+
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
```

Comment:
I don;t think this is how you're meant to set it up. I think you just need:

```python
import logging

...

logging.WARNING("")
logging.INFO("")
```

I have a feeling if you set it up by name here it stops the user being able to set the right level? But I'm not 100% sure.